x/stakeibc/keeper/community_pool.go

```diff
@@ -226,7 +226,12 @@ func (k Keeper) RedeemCommunityPoolTokens(ctx sdk.Context, hostZone types.HostZo
 
 // Using tokens in the CommunityPoolReturnIcaAddress, trigger ICA tx to fund community pool
 // Note: The denom of the passed in token has to be the denom which exists on the hostZone not Stride
-func (k Keeper) FundCommunityPool(ctx sdk.Context, hostZone types.HostZone, token sdk.Coin) error {
+func (k Keeper) FundCommunityPool(
+	ctx sdk.Context,
+	hostZone types.HostZone,
+	token sdk.Coin,
+	senderAccountType types.ICAAccountType,
+) error {
```

Comment:
!
You need to also add the address of the ICA which is sending these tokens to community pool.  This is hardcoded on line 235/240 in the MsgFundCommunityPool as the Despositor field for the way that it was before but you forgot to update it -- if you intend to call this from the withdrawal address you need to pass in the address for withdrawal ICA to that Msg type as Depositor.

Reply:
done (handled it by switching on the account type)